### DISCUSSION TITLE
Which cloud provider

### DISCUSSION THREAD

**Discussion by rajesh99** [Southclaws/storyden#177] [Q&A] [answered]
How to deploy to GCP?

**Comment by Southclaws** [comment, marked as answer]
For GCP you can deploy the pre-built docker images available here: https://github.com/Southclaws/storyden/pkgs/container/storyden

The latest image as of this comment is here: https://github.com/Southclaws/storyden/pkgs/container/storyden/248017123?tag=latest

The `fullstack` image contains the API and frontend all together which is the simplest way to get started.

These docs should be relevant for GCP:

https://cloud.google.com/run/docs/deploying

As of now, I don't have specific documentation for deploying to GCP, but I'd love to get some up on the site so if you have any feedback/issues you ran into, please share!
